### DISCUSSION TITLE
Setup mindsb with a wheels distribution folder

### DISCUSSION THREAD

**Discussion by keenkeystrokes** [mindsdb/mindsdb#2108] [Get help] [answered]
Hi team, I am new to python. How do I setup mindsdb for local development given that I have a wheels folder for it?

**Comment by ZoranPandovski** [comment, marked as answer]
Hi @vijay-garg. If you want to contribute to MindsDB the easiest would be to:

1. Fork the MindsDB repository
2. Clone the repository locally
3. Install `pip install -e .`
4. Make changes and commit them
5. Push your local branch to your fork
6. Submit a Pull Request 
7. Write a commit message
8. Make sure that the CI tests are GREEN
